src/api-reference/request/request.markdown

```diff
@@ -28,7 +28,7 @@ Name | Type | Format | Description
 -----|------|--------|------------
 `offset`	|	`string`	|	`query`	|	Starting page offset	
 `limit`	|	`Int32`	|	`query`	|	Number of records to return (default 100)	
-`user`	|	`string`	|	`query`	|	The login ID of the user who owns this Request. The user must have the Web Services Admin (Professional) or Can Administer (Standard) user role to use this parameter.	
+`user`	|	`string`	|	`query`	|	The login ID of the user who owns this Request. The user must have the Web Services Admin (Professional) or Can Administer (Standard) user role to use this parameter. Note : company-level token is not supported.
```

Comment:
Please make sure capitalization matches and remove the space after Note and before the colon.

Reply:
Thank you Charlie. I make the update asap (will simply remove the "note"), and ask for a peer reviewer on my team for a thumb up.